source/containers.tex

```diff
@@ -10351,11 +10363,12 @@
     // \ref{span.obs}, observers
     constexpr index_type size() const noexcept;
     constexpr index_type size_bytes() const noexcept;
-    constexpr bool empty() const noexcept;
+    [[nodiscard]] constexpr bool empty() const noexcept;
 
```

Comment:
Missing edit: Need to add `[[nodiscard]]` to the itemdecl of `empty()` in [span.obs] (section was wrong in paper).

Reply:
Fixed.